regression_tests/run_regression_tests.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+# If using sbatch, set the following flags
+#SBATCH -N 1
+#SBATCH -t 1:00:00
+#SBATCH -p pbatch
+#SBATCH -o sbatch.log
+#SBATCH --open-mode truncate
+# On Linux:
+# sbatch -N 1 -t 1:00:00 -p pbatch -o sbatch.log --open-mode truncate ./regression_tests/run_regression_tests.sh
```

Comment:
```suggestion
# sbatch ./regression_tests/run_regression_tests.sh
```

Reply:
> Thanks @pranav-vempati! The tests all pass on linux and mac. One small suggestion is that the squash-and-merge has resulted in some very long commit messages in the libROM history. Since this PR has almost 200 commits, it would be a good idea to replace all those commit messages with a brief summary of this PR.

 @dylan-copeland, thanks for the suggestion. I will do this before I merge this PR.